HACKING.md

```diff
@@ -156,6 +156,15 @@ done with the following command:
 
     $ prove -l t/js.t
 
+We also have a set of tests of client-side functions that can be run (after
+compiling) by opening http://localhost:5000/static/scripts/tests/web.html
```

Comment:
Yeah, re: the previous comments about all.html, it hasn't worked in a browser in years from what I can tell.  But, it's useless anyway because web.html replaced it.  (all.html is broken because it currently tries to run the Node tests; there is nothing to fix because the web tests are already run by web.html.)

So the additions here aren't wrong, but kinda confusing when read after the previous sentence about all.html, which is broken and has an unclear purpose.  I've proposed merging the Node and browser tests in https://github.com/metabrainz/musicbrainz-server/pull/2403 and made some small changes to HACKING.md there; if that looks okay we can revert the changes here so that they don't conflict.

Reply:
That's been merged, so you can rebase this PR & remove the changes here.

Reply:
Done, thanks!